.azure-pipelines/templates/copy-and-build.yml

```diff
@@ -3,17 +3,17 @@ steps:
   inputs:
     SourceFolder: '$(Agent.TempDirectory)'
     Contents: '**'
-    TargetFolder: '$(Agent.BuildDirectory)/Drop/'
+    TargetFolder: '$(DROP_PATH)/'
 
 - task: Gradle@2
   inputs:
-    gradleWrapperFile: '$(Agent.BuildDirectory)/Drop/gradlew'
-    workingDirectory: '$(Agent.BuildDirectory)/Drop/'
+    gradleWrapperFile: '$(DROP_PATH)/gradlew'
```

Comment:
there's already a trailing slash in the variable, remove this additional slash (x3)

Reply:
The current value is set to: $(Agent.BuildDirectory)/Drop
Is there an implicit slash in the variable?

Reply:
disregard, I got confused by the unused paramter